scalafix/output/src/main/scala/fix/ClientFetchTests.scala

```diff
@@ -1,3 +1,8 @@
+// Copyright 2013-2020 http4s.org
+//
+// SPDX-License-Identifier: Apache-2.0
+
+
```

Comment:
Two lines are necessary here, and I can't comment inline that they are, so I'll comment here.